### PR TITLE
Update golumns package to latest version - major bug fix

### PR DESCRIPTION
**Pull request by GrantSeltzer**
It came to my attention that `golumns` was causing an index out of bound error when trying to `ls` an empty directory. I've patched the library and this PR will update the vendoring directory to include this patch.

If someone can please confirm on their system that running `ls` on an empty directory does in fact cause an error for them before this patch and doesn't after I'd greatly appreciate it. 

Sorry about not catching this earlier!

Signed-off-by: Grantseltzer <grantseltzer@gmail.com>

### CODE REVIEW

**Comment by diwakergupta** [comment]
Confirmed that it does error out, thanks for catching that @GrantSeltzer -- need to figure out a good way of "unit" testing dbxcli to prevent these in future. Any ideas?

**Comment by GrantSeltzer** [comment]
@diwakergupta hmmm, It'll be tricky but definitely needs to be done. I guess we'd need some way of mocking the dropbox api. I can definitely start coming up with some stuff in the next couple of days :)!

File: vendor/github.com/grantseltzer/golumns/columns.go [hunk 1]
```diff
@@ -40,6 +40,9 @@ func fewestColumns(lengths []int) int {
 
 func columnLength(lengths []int) int {
 	sort.Ints(lengths)
+	if len(lengths) == 0 {
+		return 4
```
Comment (GrantSeltzer) [inline vendor/github.com/grantseltzer/golumns/columns.go]: Just re-read your comment, that's a good idea, I'll include it in the next update to golumns, working on another patch to it right now anyway.

**Comment by waits** [comment]
👍 for unit tests. I'm willing to help out as well.

**Pull request merged**